Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -32,13 +32,16 @@ struct FPTIBatchData: Codable {
         let appSwitchURL: String?
         /// UTC millisecond timestamp when a networking task started establishing a TCP connection. See [Apple's docs](https://developer.apple.com/documentation/foundation/urlsessiontasktransactionmetrics#3162615).
         /// `nil` if a persistent connection is used.
+        let buttonOrder: Int?
```

Comment:
We can make this an optional string across the board based on the slack conversation